src/host/conattrs.cpp

```diff
@@ -185,41 +167,3 @@ bool FindTableIndex(const COLORREF Color,
     }
     return false;
 }
-
-// Method Description:
-// - Get a COLORREF for the foreground component of the given legacy attributes.
-// Arguments:
-// - wLegacyAttrs - The legacy attributes to get the foreground color from.
-// - ColorTable - The array of colors to get the color from.
-// - cColorTable - The number of elements in ColorTable
-// Return Value:
-// - the COLORREF for the foreground component
-COLORREF ForegroundColor(const WORD wLegacyAttrs,
-                         _In_reads_(cColorTable) const COLORREF* const ColorTable,
-                         const size_t cColorTable)
-{
-    const byte iColorTableIndex = LOBYTE(wLegacyAttrs) & FG_ATTRS;
-
-    return (iColorTableIndex < cColorTable && iColorTableIndex >= 0) ?

```

Comment:
it was _extremely unlikely_ that we would ever get a negative value out of four bits but BY GOD we were going to make sure we didn't.

Reply:
:smile: